### ISSUE TITLE
Important information

### ISSUE THREAD

**Issue by shanear** [Irma-Response/irmarelief-static#5]
As a resident of South Florida, I want to know important information for what to do before, during and after the hurricane hits.

> One former FEMA / emergency response friend who worked Katrina said that there were some quick tips to remember (like important to take pictures of all documents, insurance info, etc.) -- we can try to connect to those folks tomorrow for additional content

- We need the content for any important information. Where should this info come from?
- How should we present this info?

**Comment by shanear** [comment]
Add a link to https://www.ready.gov/build-a-kit

**Comment by mjturtora** [comment]
https://www.fema.gov/ is on it.
suggest links to avoid repetition and focus on additional info/functionality

**Comment by shanear** [comment]
Sounds like we should just add a link there.

**Comment by mjturtora** [comment]
yup, like I've been discussing in Slack, I think a challenge to to find the line between acting as a clearinghouse and acting as an easy(ier) interface to useful information that is already on the web...as well as helping to collect information that isn't already out there.
